chassis/sonic_platform/module.py

```diff
@@ -115,6 +118,33 @@ def _get_module_eeprom_info(self):
                 eeprom_info= dict(fvs)
                 return eval(eeprom_info['eeprom_info'])
         return None
+    
+    def _update_module_hwsku_info(self):
```

Comment:
I think we can change the function name to _update_module_hwsku_info_to_supervusor.  I don't see this function is called in any place.

Reply:
Changed name. This api is called on __init__ of this class.